client/extensions/woocommerce/components/query-shipping-classes/index.js

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { fetchShippingClasses } from 'woocommerce/state/sites/shipping-classes/actions';
+import { areShippingClassesLoaded } from 'woocommerce/state/sites/shipping-classes/selectors';
+
+class QueryShippingClasses extends React.Component {
+	constructor() {
```

Comment:
I know there's some inconsistency here but might it be best to fetch from `componentDidMount`?

Reply:
Technically the constructor is called a bit earlier than `componentDidMount`, however in this case I agree that it looks better (I avoid overwriting the constructor whenever possible), so I moved the fetching.